Lapka.Pets.Infrastructure/Queries/Handlers/GetPetsByRaceHandler.cs

```diff
@@ -22,14 +20,22 @@ public GetPetsByRaceHandler(IMongoRepository<PetDocument, Guid> mongoRepository)
         {
             _mongoRepository = mongoRepository;
         }
+
         public async Task<IEnumerable<PetBasicDto>> HandleAsync(GetPetsByRace query)
         {
             if (string.IsNullOrEmpty(query.Race))
                 throw new PetsNotFoundException(query.Race);
+
+            IEnumerable<PetDocument> pets = await _mongoRepository.FindAsync(x =>
+                x.Race.ToUpper() == query.Race.ToUpper());
+
+            if (string.IsNullOrEmpty(query.Latitude) || string.IsNullOrEmpty(query.Longitude))
```

Comment:
Może by do query już dodać location jakiś, wtedy by tego nie trzeba było wszędzie sprawdzać

Reply:
User nie zawsze poda lokacje (bo np. nie udostepnil swojej lokalizacji aplikacji).

Reply:
Chodzi mi o to, że może tą logikę da się jakoś przenieść np do budowy dtosa, tak żeby tego nie musieć powtarzać.